pkg/kv/kvserver/batcheval/cmd_add_sstable.go

```diff
@@ -100,9 +100,17 @@ func declareKeysAddSSTable(
 	}
 }
 
-// EvalAddSSTable evaluates an AddSSTable command.
-// NB: These sstables do not contain intents/locks, so the code below only
-// needs to deal with MVCCKeys.
+// addSSTableCapacityRemainingLimit is the fraction of remaining store capacity
+// under which addsstable requests are rejected.
+var addSSTableCapacityRemainingLimit = settings.RegisterFloatSetting(
+	settings.SystemOnly,
+	"kv.bulk_io_write.min_capacity_remaining_fraction",
+	"remaining store capacity fraction below which an addsstable request is rejected",
+	0.05,
```

Comment:
should this be 0 for a backport to an already released version i.e. off by default and require opt-in? or do we think the value of preventing (individual) node crashes is enough to justify a potential change in behavior for bulk jobs?

Reply:
I'd lean towards setting it to 0, since that's the more conservative option and also the backport policy.

Reply:
Done.